### ISSUE TITLE
Some translations are not displayed, menu is too small (forrussian translation)

### ISSUE THREAD

**Issue by artemdudkin** [verdaccio/website#113]
Look at numbers at pictures:
1. Menu is too small - logo and text overlap each other.
2. Article title displayed on english (while it has translation actually)
3. Left sub-menus displayed on english
4. Footer displayed on english

![verdaccio-docs-translate-errors](https://user-images.githubusercontent.com/8177502/57844567-0068fd80-77d9-11e9-97e3-cfd4b7c25c9f.png)
![verdaccio-docs-translate-errors-2](https://user-images.githubusercontent.com/8177502/57844582-07900b80-77d9-11e9-8411-eb9582b300a9.png)

**Comment by juanpicado** [comment]
> Look at numbers at pictures:
> 
> 1. Menu is too small - logo and text overlap each other.

This is a Docusaurus bug, we rely on their updates. I'd rollback locally and do some regression.

> 2. Article title displayed on english (while it has translation actually)

It seems to be a bug at Docusaurus, same in other languages.

> 3. Left sub-menus displayed on english

It seems to be a bug at Docusaurus, same in other languages.

> 4. Footer displayed on english

Text is hardcoded, this is an issue in our side.

> 
> ![verdaccio-docs-translate-errors](https://user-images.githubusercontent.com/8177502/57844567-0068fd80-77d9-11e9-97e3-cfd4b7c25c9f.png)
> ![verdaccio-docs-translate-errors-2](https://user-images.githubusercontent.com/8177502/57844582-07900b80-77d9-11e9-8411-eb9582b300a9.png)

**Comment by sumitkharche** [comment]
@juanpicado Do we need to fix only point 4 against this issue?

**Comment by DanielRuf** [comment]
> @juanpicado Do we need to fix only point 4 against this issue?

Yes, we have to check 1 - 3 and see if there is a solution in Docusaurus. Point 4 can be solved by us.

**Comment by juanpicado** [comment]
I would wait for v2.

**Comment by sumitkharche** [comment]
> I would wait for v2.

When v2 will be released?
